depmapomics/dm_omics.py

```diff
@@ -814,7 +928,9 @@ async def mutationPostProcessing(
             print("saving somatic svs")
             mergedsvs_pr.to_csv(folder + "svs_profile.csv", index=False)
 
-    merged = wgsmutations_pr.append(wesmutations_pr).reset_index(drop=True)
```

Comment:
(see above) `merged` here is the profile-id level file that later gets subsetted and uploaded to the portal. so we might want to do the maf column renaming on this file instead of the sequencing-id level one. happy to go over this process in more detail in our check-in

Reply:
yes, I relocated the code to output a MAF with profile-id.